### PR TITLE
DCS-94 renaming incident id to report id

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/services/statementService.js [hunk 1]
```diff
@@ -9,28 +9,28 @@ module.exports = function createIncidentService({ incidentClient }) {
...
+    const statement = await getStatement(username, reportId, StatementStatus.PENDING)
     const errors = statementConfig.validate ? validate(statementConfig.fields, statement, true) : []
     return errors
   }
 
-  const saveStatement = (userId, incidentId, statement) => {
-    logger.info(`Saving statement for user: ${userId} and incident: ${incidentId}`)
-    return incidentClient.saveStatement(userId, incidentId, statement)
+  const saveStatement = (userId, reportId, statement) => {
+    logger.info(`Saving statement for user: ${userId} and incident: ${reportId}`)
```
Comment (Nimphal) [inline server/services/statementService.js]: Should this say 'and report'?

File: server/services/statementService.js [hunk 2]
```diff
@@ -9,28 +9,28 @@ module.exports = function createIncidentService({ incidentClient }) {
...
+    return incidentClient.saveStatement(userId, reportId, statement)
   }
 
-  const submitStatement = (userId, incidentId) => {
-    logger.info(`Submitting statement for user: ${userId} and incident: ${incidentId}`)
-    return incidentClient.submitStatement(userId, incidentId)
+  const submitStatement = (userId, reportId) => {
+    logger.info(`Submitting statement for user: ${userId} and incident: ${reportId}`)
```
Comment (Nimphal) [inline server/services/statementService.js]: Same as above, should this say 'and report'?

File: server/views/pages/submitted.html [hunk 3]
```diff
@@ -24,7 +24,7 @@
     {{
       govukButton({
         text: 'Continue to statement',
-        href: '/incidents/' + incidentId + '/statement',
+        href: '/incidents/' + reportId + '/statement',
```
Comment (Nimphal) [inline server/views/pages/submitted.html]: I'm guessing you're not changing the url just yet?

Comment (andrewrlee) [reply server/views/pages/submitted.html]: no, in a future PR

File: server/services/statementService.js (hunk 2 above)
Comment (andrewrlee) [reply server/services/statementService.js]: these are both covered in the next PR...

File: server/services/statementService.js (hunk 1 above)
Comment (andrewrlee) [reply server/services/statementService.js]: these are both covered in the next PR...

**Pull request merged**